hax/hax/queue/__init__.py

```diff
@@ -51,11 +51,14 @@ def payload_process(self, msg: str) -> None:
             'M0_HA_MSG_NVEC': self.handle_device_state_set,
             'STOB_IOQ_ERROR': self.handle_ioq_stob_error,
         }
-        if msg_type not in handlers:
+        if msg_type in handlers:
+            handlers[msg_type](payload)
+        elif msg_type == 'HA_NOTIFY':
+            self.handle_ha_notify(payload)
```

Comment:
Why can't we add 'HA_NOTIFY' to handlers dictionary? I believe `self.handle_ha_notify` has the same signature to other handlers.. And thus no extra 'elif' would be needed.

Reply:
Reverted changes.